### DISCUSSION TITLE
User Interface

### DISCUSSION THREAD

**Discussion by bhavaniravi** [bhavaniravi/kubetask#8] [Ideas]
### Table view

1. User logs in to see two sections, 
    - list of running tasks(task instances), 
    - list of all tasks that are not scheduled/Ran yet(Tasks)
2. Clicking on a task will get you the list of task instances associated with the task.

### Page view

1. Clicking on a task instance will get you state and the logs of the task
2. On the task instance screen, the user will have a share button to share the logs with fellow teammates.

**Comment by Catastrophe0123** [comment]
Is there going to be authentication for the frontend? If so, what kind?

**Comment by bhavaniravi** [comment]
Good point, let me write a separate spec for authetication
